backend/src/Entity/NotificationFirebaseTokenEntity.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\NotificationFirebaseTokenEntityRepository;
+use Doctrine\ORM\Mapping as ORM;
+use Gedmo\Mapping\Annotation as Gedmo;
+
+#[ORM\Entity(repositoryClass: NotificationFirebaseTokenEntityRepository::class)]
+class NotificationFirebaseTokenEntity
```

Comment:
Let us add a new integer field "AppType" 
this will help us to know each user for any app
this value will be sent by FE when they send the notification token